pytaxonkit.py

```diff
@@ -480,6 +480,11 @@ def test_lineage(capsys):
     assert 'taxonkit reformat --lineage-field 3 --show-lineage-taxids' in err
 
 
+def test_lineage_single_taxid():
+    result = lineage([128370])
+    assert result.TaxID.iloc[0] == 128370
+
+
```

Comment:
I wasn't able to recreate the bug I observed previously with `pytaxonkit.lineage` using only a single taxid as input. In any case, I added a regression test to be sure.